src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+  const communityAccessibilityElement = (
+    <a href={props.communityAccessibilityLink}>Website Accessibility Policy</a>
+  );
+
+  /* eslint-disable max-len */
+  return (
+    <div>
+      <h1>Studio Accessibility Policy</h1>
+      <p>
+        At edX, we seek to understand and respect the unique needs and perspectives of the edX global community.  We value every course team and are committed to being a leader in expanding access to all, including course team creators and authors with disabilities.  To that end, we have adopted a {communityAccessibilityElement} and this process to ensure that course team creators and authors with disabilities are able to develop and post content on our platform via Studio.
+      </p>
+      <p>
+        Course team creators and authors who, because of their disabilities, need assistance to use Studio should take the following steps:
+      </p>
+      <ol>
+        <li>Notify us by email addressed to {emailElement}.  In order for edX to fully and promptly assess and respond to your request, please provide the following information:</li>
+        <ul>
+          <li>Your name and email address;</li>
+          <li>The edX member institution that you are affiliated with;</li>
+          <li>A brief description of the challenge or barrier to access that you are experiencing; and</li>
+          <li>How soon you need access and for how long (e.g., a planned course start date or in connection with a course-related deadline such as a final essay).</li>
+        </ul>
+        <li>Within 48 business hours of receiving your request, the edX Support Team will respond to confirm receipt and forward your request to the edX Partner Manager for your institution and the edX Website Accessibility Coordinator.</li>
+        <li>With guidance from the Website Accessibility Coordinator, edX will contact you within 5 business days to discuss your request and gather additional information from you to identify a solution.</li>
+        <li>EdX will assist you promptly and thoroughly so that you are able to create content on the CMS within your time constraints.  Such efforts may include, but are not limited to: (a) purchasing a third-party tool or software for use on an individual basis to assist your use of Studio, (b) engaging a trained independent contractor to provide real-time visual, verbal and physical assistance, or (c) developing new code to implement a technical fix.</li>
```

Comment:
Super (Grammar) Nit: it's `edX` in some places and `EdX` in others - I think it should be `edX`?